conda_forge_tick/migrators/cross_compile.py

```diff
@@ -297,3 +297,52 @@ def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> No
 
             with open("meta.yaml", "w") as f:
                 f.write("".join(new_lines))
+
+
+CRAN_BUILD_SH = """\
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+# shellcheck disable=SC2086
+${R} CMD INSTALL --build . ${R_ARGS}
+"""
+
+
+class CrossRBaseMigrator(CrossCompilationMigratorBase):
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        host_reqs = attrs.get("requirements", {}).get("host", set())
+        if "r-base" in host_reqs or attrs.get("name", "").startswith("r-"):
```

Comment:
I think this is fine, but do we use the starts with "r-" heuristic in other places in the bot?

Reply:
We use it one other spot in the license migrator. Seems fine.